assets/infrastructure-providers/infrastructure-ibmcloud.yaml

```diff
@@ -57,8 +57,8 @@ data:
           - args:
             - --leader-elect
             - --metrics-bind-addr=127.0.0.1:8080
-            - --powervs-provider-id-fmt=v2
-            - --provider-id-fmt=${PROVIDER_ID_FORMAT:=v1}
+            - --powervs-provider-id-fmt=${POWERVS_PROVIDER_ID_FORMAT:=v1}
```

Comment:
This flag will be automatically removed while generating asset during next capi version update.
As it is already [removed](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/config/manager/manager.yaml#L29-L33) in upstream